packages/primitives/radio/src/Group.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react'
+import { component, startWithType } from 'refun'
+import { RadioContext } from './context'
+import { TRadioGroup } from './types'
+
+const RadioGroup = component(
+  startWithType<TRadioGroup>()
+)(({
+  initialValue,
+  children,
+}) => {
+  const radioState = useState(initialValue)
```

Comment:
why do we need the `useState`? dosent the user just pass `initialValue` so they can set the logic from their side?

Reply:
that's the state of the group to store what's the currently selected value.
users can handle the logic on top of that, but this is basic funcionality... like HTML api